lib/common/options.js

```diff
@@ -284,6 +284,9 @@ Options.defaults = {
     name: 'layout',
     mode: 'out-in'
   },
+  assetsDir: 'assets',
```

Comment:
If it is going to be be a best practice, i think we may move all of this options to a new `dir` block:
```js
dir: {
  assets: ...
}
```

Reply:
Yes, you're right, I'm moving.